src/ResponseSender/SendResponseEvent.php

```diff
@@ -66,9 +66,9 @@ public function setContentSent()
     {
         $response = $this->getResponse();
         $contentSent = $this->getParam('contentSent', []);
-        $contentSent[spl_object_hash($response)] = true;
+        $contentSent[$responseObjectHash = spl_object_hash($response)] = true;
```

Comment:
Please extract variable assignment to a separate line

Reply:
@Xerkus done ;)